humanlayer/core/cloud.py

```diff
@@ -26,14 +26,15 @@ class HumanLayerCloudConnection(BaseModel):
     api_base_url: str | None = None
     http_timeout_seconds: int = 30
 
-    @model_validator(mode="after")  # type: ignore
-    def post_validate(self) -> None:
+    @model_validator(mode="after")
+    def post_validate(self) -> "HumanLayerCloudConnection":
```

Comment:
you can use `Self` but since you support 3.10 you'd have to start installing `typing_extensions`